packages/authentication/README.md

```diff
@@ -1,3 +1,47 @@
 # @loopback/authentication
 
 A LoopBack component for authentication support.
+
+# Overview
+ It demonstrates how to use LoopBack's user models and passport to interact with other authentication providers.
+ User can login using a passport.js strategy, which could include a third party provider.
+
+
+# Installation
+
+```shell
+npm install --save @loopback/authentication
+```
+
+# Basic use
+
+
+ ```ts
+
+   const strategy = new BasicStrategy(async (username, password) => {
+     return await findUser(username, password);
+   };
+   getAuthenticatedUser(strategy, ParsedRequest);
```

Comment:
Why are all lines indented by two spaces? Can you remove this extra indentation please?